app/views/projects/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1><%= @project.name %></h1>
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%=@project.contestants.count %></p>
```

Comment:
Similar to what we talked about in the MVC in Action class, we do not want to be doing a `count` in the view. This is AR and should be moved to the model.